src/index.ts

```diff
@@ -265,8 +265,12 @@ async function update() {
   }
   autoUpdater.on('error',  async (error) => {
     try {
-      indexWorker.webContents.postMessage("upgrade-version-failed", {downloadUrl: await getLatestVersionLink()});
-      notify(error);
+      // Make sure that GitHub is not available before showing the error message
+      const gitHubRes = await fetch("https://github.com/");
+      if (!gitHubRes.ok) {
```

Comment:
Wait, it should be `if(gitHubRes.ok)` because we only want to notify if github is accessible and we fail to download